03-Custom-Actions-In-Python/01-What-are-Actions.ipynb

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
<p>Calling out the definition of <code>sim.state</code>  in a note feels excessive</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>